From 35b5fa941236219537bee0dd9e7a94782167bc25 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Wed, 30 Sep 2020 18:18:00 +0800
Subject: [PATCH] ARM: dts: rockchip: rv1126-evb-v12: Add board irdrop config
 for venc

The output voltage is 25mV higher than the opp voltage on v10 board,
and they are almost the same on v12 board, so add 25mV for 500MHz and 600MHz.

Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
Change-Id: Idaa48447804ca156cea1e1f77bd23b669e2c489b
---
 arch/arm/boot/dts/rv1126-evb-v12.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm/boot/dts/rv1126-evb-v12.dtsi b/arch/arm/boot/dts/rv1126-evb-v12.dtsi
index 6014ba7b7550..9b6b0610acae 100644
--- a/arch/arm/boot/dts/rv1126-evb-v12.dtsi
+++ b/arch/arm/boot/dts/rv1126-evb-v12.dtsi
@@ -59,3 +59,13 @@
 &rkvenc {
 	venc-supply = <&vdd_npu_vepu>;
 };
+
+&rkvenc_opp_table {
+	/*
+	 * max IR-drop values on different freq condition for this board!
+	 */
+	rockchip,board-irdrop = <
+	     /* MHz	MHz	uV */
+		500	594	50000
+	>;
+};
-- 
2.35.3

